src/main/kotlin/lotto/domain/LottoStore.kt

```diff
@@ -1,20 +1,30 @@
 package lotto.domain
 
+import lotto.domain.dto.PurchaseLottoRequest
+import lotto.domain.dto.PurchaseLottoResponse
+
 class LottoStore(
     private val lottoGenerator: LottoGenerator
 ) {
-    fun purchaseLottoTickets(purchaseAmount: Int): List<Lotto> {
-        require(purchaseAmount >= 1000) { AMOUNT_IS_INSUFFICIENT }
+    fun purchaseLottoTickets(purchaseLottoRequest: PurchaseLottoRequest): PurchaseLottoResponse {
+        val autoLottoTicketQuantity = getAutoLottoTicketQuantity(
+            purchaseLottoRequest.purchaseAmount.amount,
+            purchaseLottoRequest.manualLottoCount
+        )
+        val autoLottoTickets = List(autoLottoTicketQuantity) { lottoGenerator.createLotto() }
 
-        return List(getLottoTicketQuantity(purchaseAmount)) { lottoGenerator.createLotto() }
+        return PurchaseLottoResponse(
+            autoLottoTicketQuantity,
+            purchaseLottoRequest.manualLottoCount.count,
+            purchaseLottoRequest.manualLottoTickets.tickets.plus(autoLottoTickets)
```

Comment:
`plus`는 `operator fun` 이기 때문에 다음과 같이 연산자로 표현할 수도 있을 것 같아요.
```suggestion
            purchaseLottoRequest.manualLottoTickets.tickets + autoLottoTickets
```